JavaFXUserInterface/src/main/java/Controllers/AssetTypeInfoController.java

```diff
@@ -161,6 +206,144 @@ public void attachEvents() {
 
     }
 
+    private void modelsButtonPressed(){
+
+        if(modelTab.getId().equals("modelTab")){
+            int nbOfAssets = assetDAO.getAssetsFromAssetTypeID(Integer.parseInt(assetType.getId())).size();
+            trainSlider.setMax(nbOfAssets);
+            trainValue.setText(String.valueOf(trainSlider.getValue()));
+            testSlider.setMax(nbOfAssets);
+            testValue.setText(String.valueOf(testSlider.getValue()));
+
+            trainSlider.valueProperty().addListener(new ChangeListener<Number>() {
+                @Override
+                public void changed(ObservableValue<? extends Number> observableValue, Number number, Number t1) {
+                    trainValue.setText(Integer.toString((int)trainSlider.getValue()));
+                    testSlider.setMax(nbOfAssets - (int)trainSlider.getValue());
+                    trainSize = (int)trainSlider.getValue();
+                }
+            });
+            testSlider.valueProperty().addListener(new ChangeListener<Number>() {
+                @Override
+                public void changed(ObservableValue<? extends Number> observableValue, Number number, Number t1) {
+                    testValue.setText(Integer.toString((int)testSlider.getValue()));
+                    trainSlider.setMax(nbOfAssets - (int)testSlider.getValue());
```

Comment:
Given the structure of our application and the processing of data, the sliders should be independent. In our current database, all assets are already identified as being either for training or for testing. This means that the assets that you use for training and for testing are not the same and should not relate to one another. The differentiation is made through the archived attribute in the asset table in the database.

Reply:
@JayChouteau, in one of our meetings we agreed to use the archived data to evaluate the models (for both training and testing).